src/Compilers/CSharp/Test/Syntax/LexicalAndXml/XmlDocCommentTests.cs

```diff
@@ -3269,6 +3269,48 @@ public void WriteDocumentationCommentXml_01()
                 );
         }
 
+        [WorkItem(47278, "https://github.com/dotnet/roslyn/issues/47278")]
+        [Fact]
+        public void NormalizeWhitespace()
+        {
+            var text = @"namespace Foo
+{
+    /// <summary>
+    /// This is Foo.Bar
+    /// </summary>
+    class Bar
+    {
+    }
+}";
+
+            var tree = Parse(text);
+
+            Assert.Equal(@"/// <summary>
+    /// This is Foo.Bar
+    /// </summary>
+", getDocumentationCommentString());
+
+            tree = tree.WithRootAndOptions(tree.GetRoot().NormalizeWhitespace(eol: Environment.NewLine), tree.Options);
+
+            // NormalizeWhitespace didn't change the full text, but it moved whitespace into the documentation comment
+            Assert.Equal(text, tree.ToString());
+            Assert.Equal(@"/// <summary>
+    /// This is Foo.Bar
+    /// </summary>
+    ", getDocumentationCommentString());
```

Comment:
So the whitespace on this line is the difference? I don't see a difference in whitespace on any of the other lines.

Reply:
Yes. Almost any other change would require change in the text of the tree, but that is staying the same here. I have clarified the comment.

To make it more explicit: Before the call to `NormalizeWhitespace()`, the leading trivia of the `class` token is:

```
WhitespaceTrivia: "    "
SingleLineDocumentationCommentTrivia: "/// <summary>\r\n    /// This is Foo.Bar\r\n    /// </summary>\r\n"
WhitespaceTrivia: "    "
```

After the call it changes to:

```
WhitespaceTrivia: "    "
SingleLineDocumentationCommentTrivia: "/// <summary>\r\n    /// This is Foo.Bar\r\n    /// </summary>\r\n    "
```

Reply:
Thanks. Sorry for being such a pain about this: I wanted to make sure that the test fully spelled out the issue for future me :)

Reply:
@333fred No need to apologize, I understand that goal.